app/javascript/components/jewel/component.spec.js

```diff
@@ -0,0 +1,30 @@
+import { mountedComponent, screen } from "test-utils";
+
+import Jewel from "./component";
+
+describe("<Jewel /> components/jewel", () => {
+  it("renders a <Jewel /> component", () => {
+    const newProps = {
+      icon: <></>,
+      isTransparent: false,
+      className: "MuiSvgIcon-root",
+      value: ["text"]
+    };
+
+    mountedComponent(<Jewel {...newProps} />);
+    expect(screen.getByText((content, element) => element.tagName.toLowerCase() === "svg")).toBeInTheDocument();
+    expect(screen.getByText((content, element) => element.tagName.toLowerCase() === "circle")).toBeInTheDocument();
+  });
+
+  it("renders error and alert canvas", () => {
+    const newProps = {
+      icon: <></>,
+      isTransparent: false,
+      className: "MuiSvgIcon-root",
+      value: ["text"]
+    };
+
+    mountedComponent(<Jewel {...newProps} />);
+    expect(screen.getByText("text")).toBeInTheDocument();
```

Comment:
Add a test id (called something like error-icon) to the <Circle /> component in the Jewel component and check for that.